hack/addons/update-aws-cpi.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly SCRIPT_DIR
+
+# shellcheck source=hack/common.sh
+source "${SCRIPT_DIR}/../common.sh"
+
+AWS_CPI_VERSION=$1
+
+if [ -z "${AWS_CPI_VERSION:-}" ]; then
+  echo "Missing argument: AWS_CPI_VERSION"
+  exit 1
+fi
+
+ASSETS_DIR="$(mktemp -d -p "${TMPDIR:-/tmp}")"
+readonly ASSETS_DIR
+trap_add "rm -rf ${ASSETS_DIR}" EXIT
+helm repo add aws-cloud-controller-manager https://kubernetes.github.io/cloud-provider-aws
+
+CHART_VERSION=""
+if [ "${AWS_CPI_VERSION}" = "1.27.1" ]; then
+  CHART_VERSION="0.0.8"
+fi
+
+export AWS_CPI_FROM_HELM_FILE="${ASSETS_DIR}/aws-cpi-from-helm-${AWS_CPI_VERSION}.yaml"
+helm template aws-cloud-controller-manager aws-cloud-controller-manager/aws-cloud-controller-manager --version="${CHART_VERSION}" >> "${AWS_CPI_FROM_HELM_FILE}"
```

Comment:
Can we do this with `kustomize` directly like we [already do for NFD](https://github.com/d2iq-labs/capi-runtime-extensions/blob/349bf97cf3aba491f782db694cfbdbd1a4802048/hack/addons/kustomize/nfd/kustomization.yaml.tmpl#L13-L22)?